src/DynamoApplications/StartupUtils.cs

```diff
@@ -180,24 +190,68 @@ public static void PreloadShapeManager(ref string geometryFactoryPath, ref strin
         /// <summary>
         ///if we are building a model for CLI mode, then we don't want to start an updateManager
         ///for now, building an updatemanager instance requires finding Dynamo install location
-        ///which if we are running on mac os or *nix will use different logic then SandboxLookup 
+        ///which if we are running on mac os or *nix will use different logic then SandboxLookup
+        /// <paramref name="hostName"/> Dynamo variation identified by host.</param>
         /// </summary>
-        private static IUpdateManager InitializeUpdateManager()
+        private static IUpdateManager InitializeUpdateManager(string hostName)
         {
             var cfg = UpdateManagerConfiguration.GetSettings(new SandboxLookUp());
-            var um = new Dynamo.Updates.UpdateManager(cfg);
+            var um = new Dynamo.Updates.UpdateManager(cfg)
+            {
+                HostName = hostName
```

Comment:
Usually in process integration will set host version as well but since the launcher here is launching Dynamo out of process so no need to set that